### PR TITLE
fix(DB/Spawning/Pooling/Pathing) Devilsaurs, Un'goro

### PR DESCRIPTION
**Pull request by malcrom**
## Changes Proposed:
- Respawn, pool, and path Devilsaurs, Ironhide Devilsaurs, and Tyrant Devilsaurs  in Un'Goro crater.
- These are in three polls of two allowing 3 to be spawned at once. One east, one center, and one west in one of two paths.
- To be used with PR #10169.

## Issues Addressed:
- Closes 

## SOURCE:
Sniffing

## Tests Performed:
- Inserts
- tested spawning and pathing.

## How to Test the Changes:
1. .go c 24181 or .go c 24182 one will be spawned and pathing
2. .go c 24183 or .go c 24184 one will be spawned and pathing
3. .go c 24185 or .go c 24186 one will be spawned and pathing

### CODE REVIEW

**Review by acidmanifesto — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1642128832605724154.sql [hunk 1]
```diff
@@ -0,0 +1,466 @@
...
+(24184,6498,6499,6500,1,0,0,1,1,0,-7780.711,-1447.6191,-270.79898,2.897246599197387695,300,0,0,1,0,2,0,0,0,'',0),
+(24185,6498,6499,6500,1,0,0,1,1,0,-6864.7144,-619.1283,-271.304,2.897246599197387695,300,0,0,1,0,2,0,0,0,'',0),
+(24186,6498,6499,6500,1,0,0,1,1,0,-6513.4663,-749.08813,-270.07254,2.897246599197387695,300,0,0,1,0,2,0,0,0,'',0);
+
+SET @POOL := 388;
+DELETE FROM `pool_template` WHERE `entry` IN (@POOL);
+INSERT INTO `pool_template` (`entry`, `max_limit`, `description`) VALUES
+(@POOL, 1, 'Un''Goro Develsaur East Pool');
+DELETE FROM `pool_creature` WHERE `guid` IN (24181, 24182);
+INSERT INTO pool_creature (guid, pool_entry, chance, description) VALUES
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1642128832605724154.sql:22]: ¨backticks

File: data/sql/updates/pending_db_world/rev_1642128832605724154.sql [hunk 2]
```diff
@@ -0,0 +1,466 @@
...
+(24181, @POOL, 0, 'Un''Goro Develsaur East Pool 1 of 2'),
+(24182, @POOL, 0, 'Un''Goro Develsaur East Pool 2 of 2');
+
+SET @POOL := 389;
+DELETE FROM `pool_template` WHERE `entry` IN (@POOL);
+INSERT INTO `pool_template` (`entry`, `max_limit`, `description`) VALUES
+(@POOL, 1, 'Un''Goro Develsaur Center Pool');
+DELETE FROM `pool_creature` WHERE `guid` IN (24183, 24184);
+INSERT INTO pool_creature (guid, pool_entry, chance, description) VALUES
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1642128832605724154.sql:31]: same here

File: data/sql/updates/pending_db_world/rev_1642128832605724154.sql [hunk 3]
```diff
@@ -0,0 +1,466 @@
...
+(24183, @POOL, 0, 'Un''Goro Develsaur Center Pool 1 of 2'),
+(24184, @POOL, 0, 'Un''Goro Develsaur Center Pool 2 of 2');
+
+SET @POOL := 390;
+DELETE FROM `pool_template` WHERE `entry` IN (@POOL);
+INSERT INTO `pool_template` (`entry`, `max_limit`, `description`) VALUES
+(@POOL, 1, 'Un''Goro Develsaur West Pool');
+DELETE FROM `pool_creature` WHERE `guid` IN (24185, 24186);
+INSERT INTO pool_creature (guid, pool_entry, chance, description) VALUES
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1642128832605724154.sql:40]: here too

**Review by Kitzunu — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/rev_1642128832605724154.sql (hunk 3 above)
Comment (malcrom) [reply data/sql/updates/pending_db_world/rev_1642128832605724154.sql:40]: Works fine

File: data/sql/updates/pending_db_world/rev_1642128832605724154.sql (hunk 1 above)
Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1642128832605724154.sql:22]: \`pool_creature\`

**Review by Kitzunu — APPROVED** [review]

**Comment by pklloveyou** [comment]
https://github.com/azerothcore/azerothcore-wotlk/pull/10079 Check to see here? Guid overlaps

**Comment by acidmanifesto** [comment]
> #10079 Check to see here? Guid overlaps

corrected in https://github.com/azerothcore/azerothcore-wotlk/pull/10194

**Pull request merged**